frameworks/C/octane/src/program.cpp

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <uv.h>
+#include <stdbool.h>
+#include <octane.h>
+#include "responders/sds_responder.hpp"
+
+void timer_callback(uv_timer_t* timer);
+
+void on_request(http_connection* connection, http_request** requests, int number_of_requests);
+
+char* current_time;
+uv_timer_t timer;
+
+int main(int argc, char *argv[]) {
+    http_listener* listener = new_http_listener();
+    uv_timer_init(listener->loop, &timer);
+    uv_timer_start(&timer, timer_callback, 0, 500);
+
+    begin_listening(listener, "0.0.0.0", 8000, false, 4, 128, NULL, NULL, NULL, on_request);
+
+    printf("Listening...\n");
+}
+
+void on_request(http_connection* connection, http_request** requests, int number_of_requests) {
+    write_batch* batch = create_write_batch(number_of_requests);
+
+    for (int i=0; i<number_of_requests; i++) {
+        if (requests[i]->path[1] == 'p') {
+            create_plaintext_response_sds(batch);
+        } else if (requests[i]->path[1] == 'j') {
```

Comment:
I'll agree with @simongui with "everyone else is doing it" however, I will point out that someday (maybe soon) we may have another p* route, or j* route that's part of the test suite and part of the verification _may be_ testing that those routes exist.

That being said, I'm fine with this for now. @simongui does this give you any performance gains over `requests[i]->path = 'plaintext'`? Just curious.

I'm sorry if it seems like we're picking on you. It's not the intention. The benchmarks constantly need to improve and evolve and I totally appreciate your discussion in all of it.